controllers/users.js

```diff
@@ -1,3 +1,5 @@
+const axios = require("axios");
```

Comment:
any specific reason for using axios over fetch? we have a utility defined as fetch which internally uses axios

Reply:
Please refer to this discussion here https://github.com/Real-Dev-Squad/website-backend/pull/1044#discussion_r1193678416